### ISSUE TITLE
Missing make-mixins for semantic webdevelopment

### ISSUE THREAD

**Issue by duness** [jgthms/bulma#1402]
This is about Bulma and a feature wich.
Please excuse my englisch.

I am currently working with Contao and I am unable to archive a good workflow. In short: I am missing mixins to make semantic and contao-predefined Elements to work with Bulma.
In Contao a text element contains a header and text. When I type the header in the backend the header is written as < h1 >. That means I cannot give it an extra class like "title". Sure I can use @extend but it's a mess with grids. And at the end, not working properly. I am missing bootstrap-like mixins like 

```
section {
    @include make-row();
    article {
        @include make-lg-column(2);
    }
}
```

Is this something we can expect in the future? Most CMS just generate their own classes. That would not be a problem if I can easily use mixins in the css to style the grid.

**Comment by jgthms** [comment]
This would be quite some work. I’m not sure if it will be useful for all Bulma users. Are you sure there’s no workaround to add classes?

**Comment by angelocordon** [comment]
Should this issue just be merged with #33?

**Comment by jgthms** [comment]
Indeed @angelocordon.
